### PR TITLE
Feature/xc3 most expsive service irfan

### PR DESCRIPTION
**Pull request by muhammad-irfan-xgrid**
Issue: [Fix data display issue for the most expensive service on Grafana dashboard](https://github.com/XgridInc/xc3/issues/120).

[PS-1]

- Updated the query for most expensive service data display on grafana Dashboard. 

- Updated code for lambda function: cost of most expensive services.

### CODE REVIEW

File: src/expensive_services_detail/cost_metrics_of_expensive_services.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ def lambda_handler(event, context):
         for resource in top_5_resources:
             resourcedata = {
                 "Account": account_detail,
-                "Region": f"{region} ({region_names.get(region, 'unknown region name')})",
+                "Region": f"{region}-{region_names.get(region, 'unknown region name')}",
```
Comment (yaseenzafar123) [inline src/expensive_services_detail/cost_metrics_of_expensive_services.py:176]: @muhammad-irfan-xgrid Seeing the code, looks like issue is resolved, but can you also paste a screenshot from the dashboard for the verification?

File: custom_dashboard/grafana_dashboards/home-dashboard.json [hunk 2]
```diff
@@ -415,7 +415,7 @@
           },
           "editorMode": "builder",
           "exemplar": false,
-          "expr": "Expensive_Services_Detail{job=\"Most_Expensive_Services\", region=\"$region\", account=\"$Account\"}",
```
Comment (yaseenzafar123) [inline custom_dashboard/grafana_dashboards/home-dashboard.json:418]: It looks like you identified this to be a problem. Can you explain why was it so?
image

Comment (muhammad-irfan-xgrid) [reply custom_dashboard/grafana_dashboards/home-dashboard.json:418]: In **cost_metrics_of_expensive_services** Lambda function we are pushing metrics with job = Acount_id. This is why we are not able to get proper data with query we were using before. 

image

File: src/expensive_services_detail/cost_metrics_of_expensive_services.py (hunk 1 above)
Comment (muhammad-irfan-xgrid) [reply src/expensive_services_detail/cost_metrics_of_expensive_services.py:176]: Here it is. 
image

**Review by samanXgrid — APPROVED** [review]
Good work @muhammad-irfan-xgrid

**Comment by yaseenzafar123** [comment]
LGTM

**Review by yaseenzafar123 — APPROVED** [review]

**Review by mfaaraz — APPROVED** [review]
LGTM

**Pull request merged**